pkgs/applications/window-managers/i3/workstyle.nix

```
@@ -5,16 +5,26 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "workstyle";
-  version = "unstable-2021-05-09";
+  version = "unstable-2023-08-23";
 
   src = fetchFromGitHub {
     owner = "pierrechevalier83";
     repo = pname;
-    rev = "f2023750d802259ab3ee7d7d1762631ec157a0b1";
-    sha256 = "04xds691sw4pi2nq8xvdhn0312wwia60gkd8b1bjqy11zrqbivbx";
+    rev = "8bde72d9a9dd67e0fc7c0545faca53df23ed3753";
+    sha256 = "sha256-yhnt7edhgVy/cZ6FpF6AZWPoeMeEKTXP+87no2KeIYU=";
   };
 
-  cargoSha256 = "0xwv8spr96z4aimjlr15bhwl6i3zqp7jr45d9zr3sbi9d8dbdja2";
+  cargoLock = {
+    lockFile = ./Cargo.lock;
```

```suggestion
    lockFile = ./workstyle-Cargo.lock;
```
I'm not familiar with the i3 ecosystem, but there are many subpackages in the i3 directory.
Could you change this to something like workstyle-Cargo.lock, as other packages may also need Cargo.lock?

---

Good idea - file is moved.